.github/workflows/create-quantic-package.yml

```diff
@@ -45,26 +46,10 @@ jobs:
       - name: Create package version
         working-directory: ./packages/quantic
         run: |
-          echo "${{ secrets.SFDX_AUTH_PACKAGE_JWT_KEY }}" > server.key
+          echo "${{ secrets.SFDX_AUTH_JWT_KEY }}" > ${{ vars.SFDX_AUTH_JWT_KEY_FILE }}
           npx --no-install ts-node scripts/build/create-package.ts --remove-translations --ci
-          rm server.key
+          rm ${{ vars.SFDX_AUTH_JWT_KEY_FILE }}
         env:
-          SFDX_AUTH_JWT_USERNAME: sfdc.integration.devv2.hub@coveo.com
-          SFDX_AUTH_JWT_KEY_FILE: server.key
-          SFDX_AUTH_CLIENT_ID: ${{ secrets.SFDX_AUTH_PACKAGE_CLIENT_ID }}
-
-  report-status:
```

Comment:
As discussed with @erocheleau, the GitHub Slack integration has made this job obsolete so we can now remove it.

TODO: delete the SLACK_WEBHOOK_CREATE_QUANTIC_PACKAGE repository secret